### PR TITLE
TM-4575, add vice positions to mock

### PR DESCRIPTION
**Pull request by M00staff**
image

image

image

image

### CODE REVIEW

File: src/services/positions.js [hunk 1]
```diff
@@ -128,6 +128,39 @@ async function get_position_by_pos_num(query) {
...
+async function get_vice_position_by_pos_seq_num(query) {
+  const filterArg = query['rp.filter'].split('|')
+  const pos_seq_num = filterArg[2]
+  const filterSeqNum = pos_seq_num.split(',')
+
+  const data = await Positions.query(qb => {
+    qb.where('pos_seq_num', 'IN', filterSeqNum); // for now, can only filter by pos_seq_num
+  }).fetchAll({
+    withRelated: ['assignments', 'assignments.employee'],
+    require: false,
```
Comment (pveland621) [inline src/services/positions.js:140]: just a general question - what do you use to look at our mock tables?

**Comment by pveland621** [comment]
this looks good to me! seems to be a pretty standard setup for a new mock EP

File: src/services/positions.js (hunk 1 above)
Comment (M00staff) [reply src/services/positions.js:140]: DBeaver!

**Review by pveland621 — APPROVED** [review]
nice work!

**Pull request merged**